### ISSUE TITLE
Feature Request: access to a replicas IP address

### ISSUE THREAD

**Issue by gurkanguran** [microsoft/azure-container-apps#531]
Hi everyone,

I am using the Azure container apps .NET SDK to manage the tags on the container app resource. I need to be able to access the IP address of the replica for my use case. However, none of the classes has such property as far as I see in this SDK.

Is there any other way to access it or maybe I missed some class that has the IP address data of the replica?

**Comment by howang-ms** [comment]
@gurkanguran, we don't support access the replica by IP address. The container app/revision can be access by FQDN or Dapr. (https://learn.microsoft.com/en-us/azure/container-apps/connect-apps?tabs=bash) 

Can you please share more detail of your scenario? Why do you need to have the replica IP address?

**Comment by gurkanguran** [comment]
@howang-ms I was implementing a ProtoActor cluster provider for Azure Container Apps. And for members(replicas) in the cluster to communicate, they need to know each other's addresses. I needed to access and provide that info to the cluster.

I checked Orleans implementation as replica IP addresses are also needed for Orleans cluster implementation and i saw that they access it from NetworkInterfaces. So I applied the same logic in my solution and solved it like that. 

However, it would still be better to have access to the replica IP addresses directly in Azure Container Apps SDK.
